### PR TITLE
[IO-822] add a file size conversion tool.

### PR DESCRIPTION
**Pull request by hellozrh**
File size conversion tool, supporting mutual conversion of TB, GB, MB, KB, Byte, etc.,as well as converting the size of numeric types into human-readable strings. #https://issues.apache.org/jira/browse/IO-822 

 The tool can convert any size unit into a human readable version, and support file size unit conversion to and from each other. The implementation method refers to TimeUnit, adopts the feature of enumeration, simplifies the code, and has high efficiency.

### CODE REVIEW

File: src/main/java/org/apache/commons/io/FileSizeUnit.java [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+ * File size conversion tool, supporting mutual conversion of TB, GB, MB, KB, Byte, etc.,
+ * as well as converting the size of numeric types into human-readable strings.
+ * @author hellozrh
+ * @date 2023-11-19
+ */
+public enum FileSizeUnit {
+    /**Byte*/
+    BYTE {
+        @Override
+        public long toByte(long duration) {
```
Comment (michael-o) [inline src/main/java/org/apache/commons/io/FileSizeUnit.java:15]: Duration?

**Review by michael-o — CHANGES_REQUESTED** [review]
I rejected something like this in lang because it was poorly designed and this one is poorly designed as well.

File: src/main/java/org/apache/commons/io/FileSizeUnit.java (hunk 1 above)
Comment (hellozrh) [reply src/main/java/org/apache/commons/io/FileSizeUnit.java:15]: It should be length

**Review by garydgregory — CHANGES_REQUESTED** [review]
Hello @hellozrh 
Rebase on git master and run `mvn` locally before you push and fix all issues.

**Comment by michael-o** [comment]
> Hello @hellozrh Rebase on git master and run `mvn` locally before you push and fix all issues.

I closed the issue as dup.

**Comment by hellozrh** [comment]
> > Hello @hellozrh Rebase on git master and run `mvn` locally before you push and fix all issues.
> 
> I closed the issue as dup.

Discard this submission？

**Comment by michael-o** [comment]
> > > Hello @hellozrh Rebase on git master and run `mvn` locally before you push and fix all issues.
> > 
> > 
> > I closed the issue as dup.
> 
> Discard this submission？

Here yes. It should happen in Lang, but this very change is as bad as the one which was reverted in Lang.

**Comment by michael-o** [comment]
Closing, let's continue in LANG-1201.

**Pull request closed without merging**